src/program/services/downloaders/__init__.py

```
@@ -183,6 +184,16 @@ def run(
                             stream_failed_on_all_services = False
                         continue
 
+                    except InfringingTorrentException as e:
```

Ruff (0.14.14)

[error] Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)